packages/adp-tooling/src/types.ts

```diff
@@ -347,3 +398,50 @@ export interface AdpProjectData {
     reference: string;
     id: string;
 }
+
+export interface AdpCustomConfig {
+    adp: {
+        safeMode: boolean;
+        environment: ProjectType;
```

Comment:
TBD: the environment is now 'OnPrem' but our change editors still have checks for 'ABAP'

Reply:
@nikmace  Why would the change editors need to know about the target environment?